CONTRIBUTING.md

```diff
@@ -36,3 +36,26 @@ The following rule governs code contributions:
 * We use GitHub issues to track bugs and enhancement requests.
 
 * Please provide as much context as possible when you open an issue. The information you provide must be comprehensive enough to reproduce that issue for the assignee.
+
+## Development Setup
+
+As this is a CAP plugin, it is best to develop in the following calesi environment:
+```
+git clone --recursive https://github.com/cap-js/calesi.git for the calesi setup
```

Comment:
```suggestion
git clone --recursive https://github.com/cap-js/calesi.git
```
Might be confusing the string at the end(`for the calesi setup`) within the code snippet.